spec/01-abstract.md

```diff
@@ -1,19 +1,22 @@
 ## Problem Statement
 
-Distributed tracing is a methodology implemented by tracing tools enabling to follow, analyze and debug a transaction across multiple software components. Typically, a transaction passes more than one component which requires a transaction to be uniquely identified across all software components. Passing along this unique identification is referred to as context propagation.
+Distributed tracing is a methodology used by tracing tools to follow, analyze and debug a trace across multiple software components. Typically, a trace traverses more than one component which requires it to be uniquely identifiable across all participating systems. Passing along this unique identification is referred to as *context propagation*.
 
-Today context propagation is implemented in proprietary formats individually by each tracing tool provider. This results in siginficant interoperability problems between tracing tools than manifest in two major ways:
+Today, context propagation is implemented individually by each vendor.

```

Comment:
I prefer the original lines 5 and 6

Reply:
s/context propagation/trace context propagation/

Reply:
Changed in https://github.com/w3c/trace-context/pull/224/commits/738650b3c37799d62f3abbfa9315d0792435f46e
Likely, this needs to be changed throughout the spec then.